### DISCUSSION TITLE
Ask for confirmation when deleting environments

### DISCUSSION THREAD

**Discussion by peterkappelt** [community/community#12454] [Actions]
Hey everyone,

the dialog for deleting environments in the repository's settings looks like this right now:

![image](https://user-images.githubusercontent.com/13392434/156898420-051a88d3-f7f1-4cc8-b0d9-3221f6e1d96a.png)

I'd suggest to enhance this dialog a bit more, e.g. by including the name of the environment one's trying to delete or an additional checkbox.

I've recently wanted to delete an environment and closed the dialog again since I wanted to make sure that I _actually_ clicked the delete button on the right environment. I think a more verbose dialog would definitely improve overall user experience

**Comment by ethomson** [comment]
Thanks for the feedback, @peterkappelt - I agree that we could improve this and we'll make sure that it's on the backlog.
